<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeniorAdminWindow</class>
 <widget class="QMainWindow" name="SeniorAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedScheduler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/images/icon-512.png</normaloff>:/images/icon-512.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="menu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>20</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background-color: rgb(31, 31, 39);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>30</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>MedScheduler</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_get_users">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Get users</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_get_hospitals">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Get hospitals</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_add_hospital">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Add hospital</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_add_junior_admin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Add junior admin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>10</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
           color: rgb(255, 255, 255);
         font: 15pt 'Arial';
         text-align: left;
          padding-left: 10%;
        background-color: transparent;
        border: none;
        }
        QPushButton:hover {
          background-color: rgba(64, 64, 80, 100);
          border-radius: 10px;
        }</string>
         </property>
         <property name="text">
          <string>Edit profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>60</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>80</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page_get_users">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="get_users_table_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
     background-color: white;
    border: 1px solid gray;
    color: black;
    padding: 5px 10px;
    border-radius: 4px;
}

QPushButton:hover {
   background-color: #f0f0f0;
    color: #000;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
          </property>
          <property name="text">
           <string>Get users table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="users_table_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>814</width>
             <height>576</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_get_hospitals">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="get_hospitals_table_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
     background-color: white;
    border: 1px solid gray;
    color: black;
    padding: 5px 10px;
    border-radius: 4px;
}

QPushButton:hover {
   background-color: #f0f0f0;
    color: #000;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
          </property>
          <property name="text">
           <string>Get hospitals table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="hospitals_table_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>814</width>
             <height>576</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_add_junior_admin">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Data enter</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="last_name_label">
             <property name="text">
              <string>Last name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="last_name_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="first_name_text">
             <property name="text">
              <string>First name</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="first_name_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="middle_name_text">
             <property name="text">
              <string>Middle name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="middle_name_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="phone_number_text">
             <property name="text">
              <string>Phone number</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="phone_number_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="error_add_junior">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_junior_administrator_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
     background-color: white;
    border: 1px solid gray;
    color: black;
    padding: 5px 10px;
    border-radius: 4px;
}

QPushButton:hover {
   background-color: #f0f0f0;
    color: #000;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
          </property>
          <property name="text">
           <string>Add junior administrator</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_add_hospital">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Data enter</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="region_text">
             <property name="text">
              <string>Region</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="region_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="settlement_type_text">
             <property name="text">
              <string>Settlement type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="settlement_type_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="settlement_name_text">
             <property name="text">
              <string>Settlement name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="settlement_name_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="street_text">
             <property name="text">
              <string>Street</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="street_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="house_text">
             <property name="text">
              <string>House</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="house_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="hospital_full_name_text">
             <property name="text">
              <string>Hospital full name</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="hospital_full_name_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="id_of_junior_administrator_text">
             <property name="text">
              <string>Id of junior administrator</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="id_of_junior_administrator_form">
             <property name="styleSheet">
              <string notr="true">border: 1px solid gray;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLabel" name="error_add_hospital">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_new_hospital_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
     background-color: white;
    border: 1px solid gray;
    color: black;
    padding: 5px 10px;
    border-radius: 4px;
}

QPushButton:hover {
   background-color: #f0f0f0;
    color: #000;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
          </property>
          <property name="text">
           <string>Add new hospital</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_edit_profile">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Enter data you want to edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="last_name_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="first_name_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Middle name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="middle_name_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Phone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phone_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>New password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="new_password_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>New password again</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="new_password_again_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Current password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="current_password_line_edit">
          <property name="styleSheet">
           <string notr="true">border: 1px solid gray;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="edit_error_label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="submit_button">
          <property name="styleSheet">
           <string notr="true">QPushButton {
     background-color: white;
    border: 1px solid gray;
    color: black;
    padding: 5px 10px;
    border-radius: 4px;
}

QPushButton:hover {
   background-color: #f0f0f0;
    color: #000;
}

QPushButton:pressed {
    background-color: #e0e0e0;
}</string>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
